examples/react-redux/webpack.common.js

```diff
@@ -0,0 +1,43 @@
+const path = require("path");
```

Comment:
I don't have a strong opinion regarding this but since this is a very simple app shouldn't we use something like parcel to avoid config files and make the example more simpler

Reply:
We could, this was just what we used for Speedometer. The bundler was less of a focus point for that project. 
Since it works, I think this is not a show stopper, maybe a follow up task to update?